src/it/scala/io/iohk/ethereum/sync/RegularSyncItSpec.scala

```diff
@@ -96,4 +124,27 @@ class RegularSyncItSpec extends FreeSpecBase with Matchers with BeforeAndAfterAl
     }
   }
 
+  "A metric about mining a new block should be available" in customTestCaseResourceM(
```

Comment:
Very nice that you tested the metric 👍 (Though this was perhaps also out of necessity?)

Reply:
Yes, it was. I need to understand better what was happening. I started by modifying an existing test and then created this one that is a bit more specific. I also found a bit weird that there are no tests on metrics at all